etc/config.sample.toml

```diff
@@ -48,7 +48,8 @@ reporting-disabled = false
 ###
 
 [cluster]
-  shard-writer-timeout = "5s"
+  shard-writer-timeout = "5s" # The time within which a shard must respond to write.
```

Comment:
is this for a local shard? If so it seems this should be in the `data` section and not cluster